server/src/test/scala/manifests/paxinfo/NationalityBreakdownFromManifestSpec.scala

```diff
@@ -12,7 +12,7 @@ object NationalityBreakdownFromManifestSpec extends Specification {
     "Given a manifest with 1 passenger with a nationality of GB " +
       "Then I should get a Map of GBR to 1" >> {
 
-      val voyageManifest = manifestWithPassengerNationalities(List("GBR"))
+      val voyageManifest = manifestWithPassengerNationalities(List("GBR"), "2023-07-26")
```

Comment:
maybe extract this date to a descriptive variable so future readers know the significance of it